packages/react-table/src/components/Table/RowWrapper.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React from 'react';
```

Comment:
Similar question as above, looks like this is the only change to this sort of import statement while other files keep the `import * as React`